schemas/groups/navigation.json

```diff
@@ -293,6 +293,10 @@
       "description": "The intended destination of this trip",
       "type": "object",
       "properties": {
+      	"commonName": {
+          "description": "Common name of the Destination, eg 'Fiji', also used in ais messages",
+          "type": "string"
```

Comment:
Destination is currently a composite object with `waypoint` instead ov `value`. Fixed it making `commonName`, `eta` and `waypoint` normal value objects